stats.go

```diff
@@ -54,10 +54,21 @@ type Gauge interface {
 	Update(int64)
 }
 
+type stopwatch struct {
```

Comment:
Think this might have to be exported unfortunately, or else callers will not have access to the methods that return this type I believe.